README.md

```diff
@@ -38,21 +35,18 @@ that are too broadly scoped to lead to agreement, so be as specific as you can.
 
 **Example:** *The Barricade can be circumvented too easily with wall jumps.*
 
-## Proposal
+## Idea, that solves a problem
 
-You may propose a change addressing **an agreed upon opinion**, which you must
-cite within your issue. We will only discuss your proposal, not the opinion,
+You may propose a change addressing **at least one problem issue**, which you must
+cite within your issue. If at least one of the cited problems was accepted we will only discuss your idea, not the problem,
 and may accept it for implementation and testing if we feel that it poses a
-well rounded solution. Discussing alternative proposals within such an issue is
+well rounded solution. Discussing alternative ideas within such an issue is
 not allowed but others may suggest alterations or variants and you may adjust
-your initial proposal based on those. We will use the same flags as for
-opinions to keep track of good proposals even if we can't implement them
+your initial idea based on those. We will use the same flags as for
+opinions to keep track of good ideas even if we can't implement them
 immediately.
 
-We understand that it can be frustrating to have a pair of opinion and proposal
-at the ready and not being allowed to discuss them in concert but this
-limitation is important for us to keep track of things and allow multiple
-proposals to compete, without anyone questioning their intention.
+If none of the cited problems are accepted the idea will be closed after 30 days.
```

Comment:
Fine by me. Let's also change "accepted" to "agreed upon" (both label text and wording here).